modules/olm-understanding-operator-catalog-images.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-restricted-networks.adoc
+
+[id="olm-understanding-operator-catalog-images_{context}"]
+= Understanding Operator catalog images
+
+Operator Lifecycle Manager (OLM) always installs Operators from the latest
+version of an Operator catalog. As of {product-title} 4.3, Red Hat-provided
+Operators are distributed via Quay App Registry catalogs from
+link:https://quay.io/[quay.io].
+
+As catalogs are updated, the latest versions of Operators change, and older
+versions may be removed or altered. This behavior can cause problems maintaining
+reproducible installs over time. In addition, OLM running on a {product-title}
```

Comment:
Sentence sounds a bit awkward, maybe try, "In addition, when OLM runs on a {product-title} cluster in a restricted network environment, it is unable..."